src/integrations/theme/default-themes/hc_black.json

```diff
@@ -12,7 +12,9 @@
 		"statusBarItem.remoteBackground": "#00000000",
 		"ports.iconRunningProcessForeground": "#FFFFFF",
 		"editorWhitespace.foreground": "#7c7c7c",
-		"actionBar.toggledBackground": "#383a49"
+		"actionBar.toggledBackground": "#383a49",
```

Comment:
these are pulled directly from the original theme, these are not colors that we chose ourselves